app.py

```diff
@@ -107,6 +107,13 @@ def get_net_income(state_code, head_employment_income, spouse_employment_income=
 
 # Create Streamlit inputs for state code, head income, and spouse income.
 state_code = st.text_input("State Code", "CA")
+year_options = ["2023", "2022"]
```

Comment:
add 2021 and 2024

Reply:
we dont need a selector here. 2024 is fine

Reply:
Made Year Static and removed it from the front end
![Screenshot 2024-03-07 at 12 25 07 AM](https://github.com/PolicyEngine/us-marriage-incentive/assets/73863365/b03eb439-4f2b-4713-9720-500699243fc6)